src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -191,10 +192,23 @@ public int Compare(FileInfo x, FileInfo y)
                     }
 
 
-                    return x.FullName.CompareTo(y?.FullName);
+                    return x.Path.CompareTo(y?.Path);
```

Comment:
This comparer is specifically for use in ApiPort so that we sort according to display name. By switch to FullName instead of Path, then in memory assemblies will work. I'll have to check how it handles a null FullName, but I'd prefer for it fail in that case (ie, FullName should always be defined)